packages/dds/tree/src/feature-libraries/modular-schema/genericFieldKind.ts

```diff
@@ -186,35 +185,9 @@ export function convertGenericChange<TChange>(
 	genId: IdAllocator,
 	revisionMetadata: RevisionMetadataSource,
```

Comment:
Would be nice to remove the unused params. That may clean up more code in ModularChangeFamily